src/types.ts

```diff
@@ -440,3 +440,9 @@ interface AntiCheatReference {
   date: string
   reference: string
 }
+
+declare global {
+  interface WindowEventMap {
+    'controller-changed': CustomEvent<{ controllerId: string }>
```

Comment:
adding this event so I can send info from the controller helper to the controller component in react